test/test_net.rs

```diff
@@ -1,9 +1,9 @@
 use nix::net::if_::*;
 
-#[cfg(target_os = "linux")]
+#[cfg(any(target_os = "linux", target_os = "android"))]
```

Comment:
The conditions should be alphabetized, so "android" should come first here. Same for the following `#cfg` attribute.